### DISCUSSION TITLE
Unable to configure email notification with internal email server

### DISCUSSION THREAD

**Discussion by cedmct** [globaleaks/globaleaks-whistleblowing-software#4337] [Q&A]
Good morning,
I encountered some problem when trying to configure our internal mail server as the metod for notification.
From the log I see an errore that says: "Unable to verify validity of certificate: <X509name object '/CN=[servername]'>".
Can you help us?
Mail server is a Exchange 2016.

**Comment by evilaliv3** [comment]
Hello @cedmct 

GlobaLeaks veryfy any certificate involved in its connections, including the certificates of the mail servers it connects to.

Are you using a self signed certificate or is your mail exchange outdated and presenting an invalid certificate?
In this case you would need to add the certificate manually to the truststore of our Ubuntu/Debian system.

**Reply by cedmct** [reply]
Our Exchange server (2016 to be precise) is certified, as is GlobaLeaks, so this should not be a problem.
For testing purposes, we tried sending a test e-mail via telnet to our server and the message was sent and received correctly; I attach the screenshot of the test in this message.
Also, I leave the screenshot of the notification configuration here so you can see if we did anything wrong.
Thank you for your help.
![TestMailWhistleblowing](https://github.com/user-attachments/assets/334dfa93-1c43-4c95-b981-5c4b4064bf60)
![ScreenshotConfiguration](https://github.com/user-attachments/assets/bc9f00ec-4254-4ac7-ba1e-e5c4c2b76528)

**Comment by gaetanoToil** [comment]
Hi, I'm stepping in because I followed your discussion from the beginning having the exact same problem...have you tried using MX instead of IP in the “SMTP server address” box?
Go to https://mxtoolbox.com/ and enter your domain.
The mx should be something like: domain-com.mail.protection.outlook.com if you use an o365 SMTP server.
Try entering this instead of IP, on the whistleblowing portal, and let me know if it works.

**Comment by cedmct** [comment]
Hi, we tried as you told us but it still doesn't send any message. We also tried to use a free SMTP server but the results is still the same, we already used it with another software and it's functioning correctly.
Could you help us resolving this issue? We would like to use this software but it's more difficult that what we expected...
